src/mailmanclient/_client.py

```diff
@@ -255,6 +256,33 @@ def delete_list(self, fqdn_listname):
         response, content = self._connection.call(
             'lists/{0}'.format(fqdn_listname), None, 'DELETE')
 
+    def count_unsubscriber(self, list_identifier, start_date=None, stop_date=None):
+        if start_date:
```

Comment:
This is your main function. It seems fine at first glance but it needs comments. Can you add them? Always the more comments, the better. Of course you don't need to do things like: 

if (a > b):
#if a is larger than b

but meaningful comments like what the variables are supposed to have at a given point during the execution or what's the purpose of creating a dictionary called data, things like that.